bmap-button-plugin.v1.js

```diff
@@ -14,7 +14,9 @@ var BMAP = {
         ru: 'Добавьте ингредиенты в "Купи батон!"',
         en: 'Add ingredients to "Buy Me a Pie!"'
     },
+    url: 'https://app.buymeapie.com',
```

Comment:
Too vague variable name. What url? Let it be "web_app_url".
Also it should be possible to override this value through the config so that we can test the plugin on alpha and staging.